scripts/run-tests.sh

```diff
@@ -9,4 +9,4 @@ mkdir -p piccolo_admin/dist/css
 mkdir -p piccolo_admin/dist/js
 touch piccolo_admin/dist/index.html
 
-python -m pytest --cov=piccolo_admin --cov-report xml --cov-report html --cov-fail-under 85 -s $@
+python -m pytest tests --cov=piccolo_admin --cov-report xml --cov-report html --cov-fail-under 85 -s $@
```

Comment:
Hmm, this is a tricky one.

 I guess you added this so Pytest wouldn't also run the Playwright tests.

The only downside is we can no longer specify a specific test to run (passed in via $@).

```
./run-tests.sh tests/test_foo.py
```

Reply:
Yes, you're right. I had to add that because `pytest` runs both tests at once.

Reply:
Maybe instead of specifying 'tests', we can ignore the integration folder instead.

Something like `pytest --ignore=e2e`